web/package.json

```diff
@@ -15,10 +15,9 @@
     "lint": "eslint ./src",
     "lint-fix": "eslint ./src --fix",
     "pre-commit:check": "yarn lint",
-    "prepare": "husky install",
```

Comment:
we should keep this when the .husky folder is inside `web`

Reply:
fixed the prepare script so that it now works within a subdirectory